deploy/infra/aws/terraform/cloud-native/cluster-infra/local.tf

```diff
@@ -208,12 +213,14 @@ locals {
     "-s cp-billing-srv",
     "-env CP_BILLING_DISABLE_GS=\"true\"",
     "-env CP_BILLING_DISABLE_AZURE_BLOB=\"true\"",
-    "-env CP_BILLING_CENTER_KEY=\"billing-group\""
+    "-env CP_BILLING_CENTER_KEY=\"billing-group\"",
+    "-a| --cloud-config",
```

Comment:
quite strange syntax, I believe we need to simply specify `--cloud-config /root/cloud-pipeline/pipectl-deploy/cp_cloud_config.json`